doc/doxygen/references.bib

```diff
@@ -2159,4 +2168,4 @@ @article{zalesak1979fully
   pages={335--362},
   year={1979},
   publisher={Elsevier}
-}
+}
```

Comment:
Can you please fix the missing line break at the end of the file, such that this diff here disappears?